Extension/package.json

```diff
@@ -1353,6 +1353,16 @@
                     "type": "boolean",
                     "description": "%c_cpp.debuggers.logging.engineLogging.description%",
                     "default": false
+                  },
+                  "threadExit": {
+                    "type": "boolean",
+                    "description": "%c_cpp.debuggers.logging.threadExit.description%",
+                    "default": false
```

Comment:
why is this default `false` but the processExit `true` ? by default we should be printing all these messages and allow the user to override with `false`. In the past the `default` has been what we have it set internally as `default` if the value isn't set.

Reply:
I assumed that an app could have tons of threads exiting, so that could be spammy to show by default, while the process exiting is only supposed to occur one time?